grace/server.go

```diff
@@ -240,6 +248,7 @@ func (srv *Server) serverTimeout(d time.Duration) {
 			break
 		}
 		srv.wg.Done()
```

Comment:
Why you're still calling `srv.wg.Done()` when it's already deferred at the beginning.